util/archutil/amd64_binary.go

```diff
@@ -6,4 +6,4 @@ package archutil
 // This file is generated by running make inside the archutil package.
 // Do not edit manually.
 
-const Binaryamd64 = "\x1f\x8b\x08\x00\x00\x00\x00\x00\x02\xff\xaa\x77\xf5\x71\x63\x62\x64\x64\x80\x01\x26\x06\x3b\x06\x30\x4f\xc0\x01\xcc\x77\x80\x8a\x1b\x08\xc0\x95\x30\x38\x30\x58\x30\xb0\x30\x38\x30\xb0\x30\x30\x83\xd5\xb2\x30\x20\x03\x07\x14\xda\x01\x6a\x34\x8c\x66\x80\x9a\x03\xe2\xb2\x22\xf1\x61\xf6\xc1\x68\x1e\xa8\x30\x8c\x86\xa9\x63\x81\x62\x05\xa8\x79\x0a\x8c\x0e\xa8\x34\x54\x39\x8c\xe6\x80\xd2\x81\x4f\x4b\x52\xd8\x18\x88\x07\x30\x67\xb1\x40\xd9\x20\xb7\xba\xfb\x85\x82\xdc\x7d\x80\x05\xea\xfe\x51\x30\x0a\x46\xc1\x28\x18\x05\xa3\x60\x14\x8c\x82\x51\x30\x0a\x46\xc1\x28\x18\x05\xa3\x60\xa8\x00\x8f\xe3\x07\x6c\x40\x94\xe1\x7f\x7e\x56\x06\xbd\xe2\x8c\xe2\x92\xa2\x92\xc4\x24\x06\xbd\xbc\xfc\x92\x54\xbd\xf4\xbc\x52\xbd\x82\xa2\xfc\x82\xd4\xa2\x92\x4a\x06\xbd\x92\xd4\x8a\x12\x8a\xed\xe3\x66\x60\x60\x60\x07\x8f\x33\xa0\xf7\xdf\x51\xfb\xed\x0c\x68\xfd\x77\x18\x90\x83\xf6\xbd\xe1\x7d\x79\xf8\xb8\x01\xda\x78\x01\x03\x62\x9c\x01\x9d\xcf\x8c\xc5\x5d\x3c\x50\xfd\x2a\x04\xf4\x03\x02\x00\x00\xff\xff\x90\x57\x64\xbb\x30\x11\x00\x00"
+const Binaryamd64 = "\x1f\x8b\x08\x00\x00\x00\x00\x00\x02\xff\xaa\x77\xf5\x71\x63\x62\x64\x64\x80\x01\x26\x06\x3b\x06\x30\x4f\xc0\x01\xcc\x77\x80\x8a\x2b\x08\xc0\x95\x30\x38\x30\x58\x30\x30\x33\x38\x30\x30\x33\x30\x81\xd5\xb2\x30\x20\x03\x07\x14\xfa\x05\x94\x07\xa3\x19\xa0\xe6\x80\xf4\xb1\x22\xf1\x61\xf6\xc1\x68\x1e\xa8\x30\x0f\x9a\xbe\xc0\xa7\x25\x29\x6c\x0c\xc4\x03\x01\x86\x51\x30\x0a\x46\xc1\x28\x18\x05\xa3\x60\x14\x8c\x82\x51\x30\x0a\x46\xc1\x28\x18\x05\xa3\x60\x14\x78\x1c\x3f\x60\x03\xa2\x0c\xff\xf3\xb3\x32\xe8\x15\x67\x14\x97\x14\x95\x24\x26\x31\xe8\x95\xa4\x56\x94\x50\xc3\x7c\x6e\x68\x5f\x1f\xde\x67\x87\xf7\xf3\xd1\xfa\xf7\x0c\x88\x71\x01\x74\x3e\x33\x16\x73\x79\xa0\xfa\x05\x09\xe8\x07\x04\x00\x00\xff\xff\x6d\x7a\x0b\x2f\xe0\x10\x00\x00"
```

Comment:
What was changed?

Reply:
```
/tmp # x86_64-alpine-linux-musl-objdump -D exit_amd64_old

exit_amd64_old:     file format elf64-x86-64


Disassembly of section .note.gnu.property:

0000000000400120 <.note.gnu.property>:
  400120:	04 00                	add    $0x0,%al
  400122:	00 00                	add    %al,(%rax)
  400124:	10 00                	adc    %al,(%rax)
  400126:	00 00                	add    %al,(%rax)
  400128:	05 00 00 00 47       	add    $0x47000000,%eax
  40012d:	4e 55                	rex.WRX push %rbp
  40012f:	00 01                	add    %al,(%rcx)
  400131:	00 00                	add    %al,(%rax)
  400133:	c0 04 00 00          	rolb   $0x0,(%rax,%rax,1)
  400137:	00 01                	add    %al,(%rcx)
  400139:	00 00                	add    %al,(%rax)
  40013b:	00 00                	add    %al,(%rax)
  40013d:	00 00                	add    %al,(%rax)
	...

Disassembly of section .text:

0000000000401000 <.text>:
  401000:	48 c7 c0 3c 00 00 00 	mov    $0x3c,%rax
  401007:	48 31 ff             	xor    %rdi,%rdi
  40100a:	0f 05                	syscall
/tmp # x86_64-alpine-linux-musl-objdump -D exit_amd64_new

exit_amd64_new:     file format elf64-x86-64


Disassembly of section .text:

0000000000401000 <.text>:
  401000:	48 c7 c0 3c 00 00 00 	mov    $0x3c,%rax
  401007:	48 31 ff             	xor    %rdi,%rdi
  40100a:	0f 05                	syscall
```

I guess the updated version in bullseye is smarter.